java/test/jmri/configurexml/LoadAndStoreTestBase.java

```diff
@@ -45,12 +46,22 @@
 @RunWith(Parameterized.class)
 public class LoadAndStoreTestBase {
 
+    // allows code reuse when building the parameter collection in getFiles()
     private final File file;
-    // allows code reuse when building the parameter
-    // collection in getFiles()
+
+    public enum SaveType {
+        All, Config, Prefs, User, UserPrefs
+    }
+
+    private SaveType saveType = SaveType.Config;
 
     public LoadAndStoreTestBase(File file, boolean pass) {
```

Comment:
I would get rid of this method, and make the saveType parameter be explicitly set in every class that calls this constructor.

Reply:
Done.